src/Symfony/Component/Clock/MockClock.php

```diff
@@ -49,7 +49,12 @@ public function sleep(float|int $seconds): void
 
     public function modify(string $modifier): void
     {
-        if (false === $modifiedNow = @$this->now->modify($modifier)) {
+        try {
+            $modifiedNow = @$this->now->modify($modifier);
+        } catch (\DateMalformedStringException) {
```

Comment:
should we attach it as previous exception of the exception we throw ?

Reply:
I'm not sure: the exception doesn't contain anything useful